### ISSUE TITLE
SmartCity-Accessibility #142 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#111]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 88 lines...]
  symbol: method getServices()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[32,9] cannot find symbol
  symbol:   method setOnMapReadyHandler(<anonymous com.teamdev.jxmaps.MapReadyHandler>)
  location: class smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[67,22] no suitable method found for add(smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt,java.lang.String)
    method java.awt.Container.add(java.lang.String,java.awt.Component) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.lang.String)
    method java.awt.Container.add(java.awt.Component,int) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
    method java.awt.Container.add(java.awt.Component,java.lang.Object) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[INFO] 7 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13.435 s
[INFO] Finished at: 2016-12-27T20:05:46+00:00
[INFO] Final Memory: 26M/172M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[20,65] cannot find symbol
[ERROR] symbol:   class waitableMap
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[22,41] cannot find symbol
[ERROR] symbol: class waitableMap
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[36,34] cannot find symbol
[ERROR] symbol: method getMap()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[51,95] incompatible types: smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to smartcity.accessibility.mapmanagement.JxMapsFunctionality.extendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[46,18] cannot find symbol
[ERROR] symbol: method getServices()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[32,9] cannot find symbol
[ERROR] symbol:   method setOnMapReadyHandler(<anonymous com.teamdev.jxmaps.MapReadyHandler>)
[ERROR] location: class smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[67,22] no suitable method found for add(smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt,java.lang.String)
[ERROR] method java.awt.Container.add(java.lang.String,java.awt.Component) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.lang.String)
[ERROR] method java.awt.Container.add(java.awt.Component,int) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[ERROR] method java.awt.Container.add(java.awt.Component,java.lang.Object) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 390c3554c93392e718742f7db621ace4fc2b4dee - #94 can clear markers set

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/142/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 88 lines...]
  symbol: method getServices()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[32,9] cannot find symbol
  symbol:   method setOnMapReadyHandler(<anonymous com.teamdev.jxmaps.MapReadyHandler>)
  location: class smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[67,22] no suitable method found for add(smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt,java.lang.String)
    method java.awt.Container.add(java.lang.String,java.awt.Component) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.lang.String)
    method java.awt.Container.add(java.awt.Component,int) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
    method java.awt.Container.add(java.awt.Component,java.lang.Object) is not applicable
      (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[INFO] 7 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 12.623 s
[INFO] Finished at: 2016-12-27T20:14:20+00:00
[INFO] Final Memory: 26M/162M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[20,65] cannot find symbol
[ERROR] symbol:   class waitableMap
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[22,41] cannot find symbol
[ERROR] symbol: class waitableMap
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[36,34] cannot find symbol
[ERROR] symbol: method getMap()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[51,95] incompatible types: smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to smartcity.accessibility.mapmanagement.JxMapsFunctionality.extendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[46,18] cannot find symbol
[ERROR] symbol: method getServices()
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[32,9] cannot find symbol
[ERROR] symbol:   method setOnMapReadyHandler(<anonymous com.teamdev.jxmaps.MapReadyHandler>)
[ERROR] location: class smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/acessibility/jxMapsFunctionality/placeSearchAttempt.java:[67,22] no suitable method found for add(smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt,java.lang.String)
[ERROR] method java.awt.Container.add(java.lang.String,java.awt.Component) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.lang.String)
[ERROR] method java.awt.Container.add(java.awt.Component,int) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[ERROR] method java.awt.Container.add(java.awt.Component,java.lang.Object) is not applicable
[ERROR] (argument mismatch; smartcity.acessibility.jxMapsFunctionality.placeSearchAttempt cannot be converted to java.awt.Component)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 390c3554c93392e718742f7db621ace4fc2b4dee - #94 can clear markers set

- [Alexander Kaplan] 52f4af29fcb353775ba95f1bf6720d37068c0096 - added experiment for #29 and a glass panel for future menu



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/143/)

**Comment by kapalex** [comment]
@Kolikant

**Comment by Kolikant** [comment]
@KaplanAlexander

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
